twoslab.md

```diff
@@ -0,0 +1,42 @@
+# Two Slab Model
+![viewshieldblock701](viewshieldblock701.png)<br/></br>
+The simple two slab model geometry consists of two slabs and a cube that holds an isotropic 14 MeV source.
+#### Goals of Example
+
+  * Use the UW2 workflow
+  * Use a simple PyNE script to make materials
+  * Use a Cubit/Trelis script to create the model and prepare it for DAGMC
+  * Inject materials into the h5m faceted geometry file for transport
+  * Run dag-mcnp6 with the CAD model prepared above
+
+#### Procedure Overview
+
+  1. Create materials by running a simple python PyNE script
+  2. Create the two slab model and prepare it for DAGMC by running a Cubit/Trelis journal script
+  3. Run uwuw_preproc to inject materials into h5m faceted geometry file
+  4. Run DAG-MCNP62 using the supplied input file
+
+_Cubit is a meshing pre-processing software with CAD capabilities and Trelis is the commercially available version. It is used in this workflow to create the CAD model, assign materials to volumes, and finally export the model to the faceted DAGMC .h5m format._<br/></br>
+_Preparing and exporting the model to a faceted h5m format requires Cubit/Trelis and the DAGMC Cubit/Trelis plugin, which should be installed before starting the tutorial. Instructions are [here](https://svalinn.github.io/DAGMC/install/plugin.html)._
+
+## Procedure:
+
+### In the bash command shell type:

```

Comment:
These headings should have titles that match the numbered list of steps above so learners can see where they are in the process

Reply:
Yes, that will be more consistent.